CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [8.30.0](https://github.com/instructure/instructure-ui/compare/v8.29.0...v8.30.0) (2022-10-26)
+
+
+### Bug Fixes
+
+* **ui-code-editor:** integrate `react-codemirror2` ([1fee246](https://github.com/instructure/instructure-ui/commit/1fee246068aff15d9bdbae882e48431bd73df6a3))
```

Comment:
Can you please add a little more explanation to this?

Reply:
Sure, can you check it again?